package.json

```diff
@@ -47,8 +44,14 @@
     "webmaker-app-icons": "git://github.com/k88hudson/ionicons#teach",
     "webpack": "^1.6.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "mocha-phantomjs": "^3.5.3",
+    "phantomjs": "^1.9.16",
+    "simplecrawler": "^0.5.2",
+    "supertest": "^1.0.1"
+  },
   "scripts": {
+    "postinstall": "gulp postinstall",
```

Comment:
Oh no, the `postinstall` step is solely for deploying to heroku--devs should never have to run it (unless they're actually trying to simulate a production deployment as closely as possible). The "boot timeout error" is also only relevant to heroku and not the local dev environment--in fact generating the JS bundle shouldn't take *too* long on local dev environments because it's not optimizing/minimizing the code (that's what really makes generating the bundle take so long in production).

Hmm, I'm thinking it might actually help for you to try deploying this server to heroku yourself--that way you can observe what commands it's running, and how its way of deploying things is quite different from the way developers work on the site. (It's not great for dev/prod parity, sadly, but I can't think of any other way to do it.)

In short, heroku never runs `npm start`. As I understand it, this is roughly what happens when you push the repository to heroku:

1. Heroku checks out your repo on a build machine and runs `npm install` with `NODE_ENV=production`--so your LESS and client-side JS bundles are generated in `dist`.
2. Heroku takes a snapshot of the repo directory structure--including the `dist` and `node_modules` directory--and saves it as something called a "slug", which is basically just an archive it can easily copy to a brand-new machine to easily spin up a new web process.
3. When told to start a new web process, Heroku spins up a new server, loads the slug onto it, and follows the instructions specified in the project's `Procfile`--in this case, it runs `node app.js`.

So to "simulate" what it'd be like for Heroku to run it, I followed the same kind of procedure on my local machine.